scripts/leapfrog/playbooks/logstash-start.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This playbook needs the boiler plate license for RPC-OpenStack

Reply:
cool, I'll add it